insights/parsers/systemd/unitfiles.py

```diff
@@ -81,3 +73,88 @@ def is_on(self, service_name):
             bool: True if service is enabled, False otherwise
         """
         return self.services.get(service_name, False)
+
+    def get_service_details(self, service_name):
+        """
```

Comment:
I'm not sure this will work.  Since both `UnitFiles` and `ListUnits` inherit from the same base class they will each have the methods `is_on` and `get_service_details`, however those methods are really specific to the parser and not really applicable to all child classes.  I think it would be better to not use the base class `SystemctlServiceDetails` here since the parsers are actually different.